lib/bindConnectors/bindMessage.js

```diff
@@ -48,11 +144,15 @@ module.exports = function (message, threadneedle, connectorConfig) {
 		});
 	};
 
-	// Bind the method to the internal handler object - this is how the connector
-	// processes the inbound messages below.
-	return function (event) {
-		return when.promise(function (resolve, reject) {
+	const constructRequestUrl = (baseUrl, body, path) => {
```

Comment:
1. Why is this function inside the main function? (instead of top level)
2. Does this need to be a function? (you could just do this as a ternary condition)?
3. This is assuming the operation we're dealing with is a standard REST operation - what about the function operations or SOAP operations?
4. I'm not convinced we should be logging out the URL

Reply:
Just to note: falafel doesn't actually know a whole lot about the model.js - yes, there are some props it's aware of/uses, but generally, model.js is passed onto threadneedle, as model.js is threadneedle's domain. 
We should be careful about making falafel logic aware of the model.

Reply:
why can't we log the URL?